pulp_service/pulp_service/app/models.py

```diff
@@ -40,6 +40,11 @@ class DomainOrg(models.Model):
         settings.AUTH_USER_MODEL,
         related_name="users",
         on_delete=models.CASCADE,
+    )
+    group = models.ForeignKey(
+        Group,
+        related_name="domainorgs",
+        on_delete=models.CASCADE,
```

Comment:
I think these should be `models.SET_NULL` and similar for `user` above. The reason I think is that when we remove a user or team (aka Group) from our system, we don't want to also delete `DomainOrg` objects.